### DISCUSSION TITLE
Failed to use basicvsr_pp

### DISCUSSION THREAD

**Discussion by relic-yuexi** [open-mmlab/mmediting#2002] [How-to]
```
mkdir_or_exist(os.path.dirname(result_out_dir))
editor = MMagicInferencer('basicvsr_pp')
results = editor.infer(video=video, result_out_dir=result_out_dir)
```

get WARNING 

08/27 00:51:18 - mmengine - INFO - Loads checkpoint by http backend from path: https://download.openmmlab.com/mmediting/restorers/basicvsr/spynet_20210409-c6c1bd09.pth
Loads checkpoint by http backend from path: https://download.openmmlab.com/mmediting/restorers/basicvsr_plusplus/basicvsr_plusplus_c64n7_8x1_600k_reds4_20210217-db622b2f.pth
08/27 00:51:19 - mmengine - WARNING - Failed to search registry with scope "mmagic" in the "function" registry tree. As a workaround, the current "function" registry in "mmengine" is used to build instance. This may cause unexpected failure when running the built modules. Please check whether "mmagic" is a correct scope, or whether the registry is initialized.

By the way, if i used basicvsr, i get nothing.

```
video = './S01E01_Open_Government.mkv'
result_out_dir = './out_video/S01E01_Open_Government.mkv'
mkdir_or_exist(os.path.dirname(result_out_dir))
editor = MMagicInferencer('basicvsr')
results = editor.infer(video=video, result_out_dir=result_out_dir)
```

08/27 00:53:32 - mmengine - INFO - Loads checkpoint by http backend from path: https://download.openmmlab.com/mmediting/restorers/basicvsr/spynet_20210409-c6c1bd09.pth
Loads checkpoint by http backend from path: https://download.openmmlab.com/mmediting/restorers/basicvsr/basicvsr_reds4_20120409-0e599677.pth

and nothing happen

**Comment by Git-Digger** [comment]
hxd解决了吗,我也这样还爆显存了

**Reply by Git-Digger** [reply]
而且我是每一个model都不行不只是basicvsr++
